### PR TITLE
Revamp mirrord buttons

### LINKED ISSUE #14: Add button to submit feedback
Allow to open the feedback from from a button or command.

### PR DESCRIPTION
**Pull request by DmitryDodzin**
Change status bar buttons to be more clear on what they do with minimal footprint

closes #14

### CODE REVIEW

**Review by Razz4780 — APPROVED** [review]
Looks stunning ^^

Noticed some quirks aroud `contributions` entry in `package.json` - settings command is never shown, but I guess it's out of scope

**Comment by Razz4780** [comment]
Actually, the feedback link shows the homepage (https://mirrord.dev/feedback). Probably not what we want

**Review by eyalb181 — CHANGES_REQUESTED** [review]
The menu takes a really long time to show up when hovering - it's essentially an easter egg. Is there a way to make the menu show up faster?

**Comment by Razz4780** [comment]
> The menu takes a really long time to show up when hovering - it's essentially an easter egg. Is there a way to make the menu show up faster?

Couldn't find anything in the VS Code API. Also, this is something the user can set in their preferences. Probably out of our control

**Review by eyalb181 — APPROVED** [review]

File: src/status.ts [hunk 1]
```diff
@@ -33,6 +36,7 @@ export class MirrordStatus {
         statusBar.tooltip.appendMarkdown(`[${enabled ? 'Enabled' : 'Disabled'}](command:${toggleCommandId})`);
         statusBar.tooltip.appendText("\n\n");
         statusBar.tooltip.appendMarkdown(`\n\n[Settings](command:${settingsCommandId})`);
+        statusBar.tooltip.appendMarkdown(`\n\n[Teams Waitlist](command:${waitlistCommandId})`);
```
Comment (eyalb181) [inline src/status.ts:39]:
```suggestion
        statusBar.tooltip.appendMarkdown(`\n\n[mirrord for Teams Waitlist](command:${waitlistCommandId})`);
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**